### ISSUE TITLE
OnVolume Exception

### ISSUE THREAD

**Issue by mphilipp622** [beaverlyhillsstudios/wavesurfer-blazor-wrapper#2]
Hey, thanks for making this wrapper. I ran into an issue trying to use the OnVolume parameter on the WavesurferPlayer component. It throws an exception:

```
Uncaught (in promise) Error: System.Text.Json.JsonException: The JSON value could not be converted to System.Int32. Path: $ | LineNumber: 0 | BytePositionInLine: 3.
 ---> System.FormatException: Either the JSON value is not in a supported format, or is out of bounds for an Int32.
   at System.Text.Json.Utf8JsonReader.GetInt32()
   at System.Text.Json.Serialization.Converters.Int32Converter.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1[[System.Int32, System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, Int32& value)
   at System.Text.Json.Serialization.JsonConverter`1[[System.Int32, System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].ReadCore(Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
   --- End of inner exception stack trace ---
   at System.Text.Json.ThrowHelper.ReThrowWithPath(ReadStack& state, Utf8JsonReader& reader, Exception ex)
   at System.Text.Json.Serialization.JsonConverter`1[[System.Int32, System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].ReadCore(Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
   at System.Text.Json.Serialization.JsonConverter`1[[System.Int32, System.Private.CoreLib, Version=6.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].ReadCoreAsObject(Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
   at System.Text.Json.JsonSerializer.ReadCore[Object](JsonConverter jsonConverter, Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
   at System.Text.Json.JsonSerializer.Read[Object](Utf8JsonReader& reader, JsonTypeInfo jsonTypeInfo)
   at System.Text.Json.JsonSerializer.Deserialize(Utf8JsonReader& reader, Type returnType, JsonSerializerOptions options)
   at Microsoft.JSInterop.Infrastructure.DotNetDispatcher.ParseArguments(JSRuntime jsRuntime, String methodIdentifier, String arguments, Type[] parameterTypes)
   at Microsoft.JSInterop.Infrastructure.DotNetDispatcher.InvokeSynchronously(JSRuntime jsRuntime, DotNetInvocationInfo& callInfo, IDotNetObjectReference objectReference, String argsJson)
   at Microsoft.JSInterop.Infrastructure.DotNetDispatcher.BeginInvokeDotNet(JSRuntime jsRuntime, DotNetInvocationInfo invocationInfo, String argsJson)
```

I think it's throwing this because the volume is supposed to be a `float` type, but the OnVolume parameter is `EventCallback<int>`. I think the JS interop is having an issue when calling back to C# with the value. This is just my suspicion.

This should be a minimal reproduction. Due to CORS issues, I don't have a URL to use in the URL parameter so you'll need your own.

```csharp
<WavesurferPlayer @ref="this.audioPlayer"
                      Url="PUT YOUR AUDIO FILE HERE"
                      OnVolume="this.OnVolumeChanged"/>
<button @onclick="SetVolume">Change Volume</button>

@code{

    private WavesurferPlayer? audioPlayer = null;

    private void OnVolumeChanged(int newVol)
    {
        Console.WriteLine($"New volume = {newVol}");
    }
    private async Task SetVolume()
    {
        if (this.audioPlayer != null)
        {
            Random rand = new();
            await this.audioPlayer.SetVolume(rand.NextSingle());
        }
    }
}
```

**Comment by rstefek** [comment]
Hello, thanks for pointing that out, I am gonna check that out

**Comment by rstefek** [comment]
Fixed in release 0.2.1
